integration-tests/trait-dyn-cross-contract-calls/lib.rs

```diff
@@ -0,0 +1,110 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
A top level `README` or description of the test setup here would be quite helpful